server/flyway/sql/V51__add_user_terms_conditions_table.sql

```diff
@@ -0,0 +1,47 @@
+-- Create fam_user_terms_conditions table
+CREATE TABLE IF NOT EXISTS app_fam.fam_user_terms_conditions
+(
+    user_terms_conditions_id        bigint          GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    user_id                         bigint          NOT NULL,
+    version                         varchar(30)     NOT NULL,
+    create_user                     varchar(100)    NOT NULL,
+    create_date                     timestamp(6)    DEFAULT CURRENT_DATE NOT NULL,
```

Comment:
I think we can use create date as acceptance date?

Reply:
I feel the same, but probably there is no current need.